tajo-core-tests/pom.xml

```diff
@@ -375,7 +375,7 @@
               <forkCount>${maven.fork.count}</forkCount>
               <reuseForks>true</reuseForks>
               <trimStackTrace>false</trimStackTrace>
-              <argLine>-Xms512m -Xmx1024m -XX:MaxPermSize=152m -Dfile.encoding=UTF-8 -Dderby.storage.pageSize=1024 -Dderby.stream.error.file=/dev/null</argLine>
+              <argLine>-Xms512m -Xmx1024m -Dfile.encoding=UTF-8 -Dderby.storage.pageSize=1024 -Dderby.stream.error.file=/dev/null</argLine>
```

Comment:
With this option, Infinite GC POOL is occured during parallel test and cannot run next test. To avoid this phenomenon, I enter the parallel test without XX:MaxMetaspaceSize.
Currently, Whole test of TAJO is running without this option after version up of JAVA.